<?xml version="1.0" encoding="UTF-8"?>
<testsuites failures="15" skipped="0" tests="95" testsuite_failed="True" time="123.691786" timestamp="2020-02-19T00:41:10.000356">
 <properties>
  <property name="module" value="FixExchangeEmulator"/>
  <property name="module" value="GatewayNativeClient"/>
  <property name="python-module-path" value="ScriptKit"/>
  <property name="python-module-path" value="Scripts/Gateway/SystemAreas/OrderProcessing/PythonModules"/>
  <property name="python-module-path" value="Scripts/Gateway/PythonModules"/>
  <property name="python-module-path" value="Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders/MultipleActions/PythonModules"/>
  <property name="python-module-path" value="Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders/PythonModules"/>
  <property name="environment-config-repository" value="XML:Scripts/Gateway/EnvConfigs/GatewayTestingEnvironmentConfigRepository.xml"/>
  <property name="modules-cfg-instance" value="SF_ESR1"/>
  <property name="scripts-cfg-instance" value=""/>
  <property name="threads-count" value="1"/>
  <property name="path" value="Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders"/>
  <property name="host" value="k12sgwcendev31"/>
 </properties>
 <testsuite failures="15" name="CompoundOrders" skipped="0" tests="95" testsuite_failed="True" time="123.691786" timestamp="2020-02-19T00:41:10.000356">
  <testsuite failures="0" name="Brackets" skipped="0" tests="4" testsuite_failed="False" time="2.7344619999999997" timestamp="2020-02-19T00:41:21.891392">
   <testcase name="ModifyBracketOCOLegAfterFillOPOPriLeg" status="run" time="0.250204" timestamp="2020-02-19T00:41:22.078902"/>
   <testcase name="OPO_L0_OCO_ModL2FlL0FlL0FlL2" status="run" time="0.281505" timestamp="2020-02-19T00:41:22.438555"/>
   <testcase name="OPO_L1_OCO_FillL1FillL3" status="run" time="0.250005" timestamp="2020-02-19T00:41:22.828903"/>
   <testsuite failures="0" name="CancelAll" skipped="0" tests="1" testsuite_failed="False" time="1.393777" timestamp="2020-02-19T00:41:23.125803">
    <testcase name="BracketSuspendedCancel" status="run" time="0.171884" timestamp="2020-02-19T00:41:24.235213"/>
   </testsuite>
  </testsuite>
  <testsuite failures="0" name="MultipleActions" skipped="0" tests="1" testsuite_failed="False" time="2.437572" timestamp="2020-02-19T00:41:24.672736">
   <testcase name="SuspendActions" status="run" time="2.203179" timestamp="2020-02-19T00:41:24.828992"/>
  </testsuite>
  <testsuite failures="0" name="OCO" skipped="0" tests="24" testsuite_failed="False" time="19.344387" timestamp="2020-02-19T00:41:27.157193">
   <testcase name="AddTwoNewOrders" status="run" time="0.171878" timestamp="2020-02-19T00:41:27.297818"/>
   <testcase name="DurationModificationForOCO" status="run" time="1.078144" timestamp="2020-02-19T00:41:27.579087"/>
   <testcase name="FLegFilledPartialySLegFilledFully" status="run" time="0.126018" timestamp="2020-02-19T00:41:28.766618"/>
   <testcase name="FLegFilledSLegCancelled" status="run" time="0.109377" timestamp="2020-02-19T00:41:28.985364"/>
   <testcase name="Place4LegsCancel3Legs" status="run" time="0.267696" timestamp="2020-02-19T00:41:29.157251"/>
   <testcase name="Place4LegsCancelFirstLeg" status="run" time="0.17176" timestamp="2020-02-19T00:41:29.501127"/>
   <testcase name="PlaceFromAccountTraderNotAutorized" status="run" time="0.046903" timestamp="2020-02-19T00:41:29.766657"/>
   <testcase name="PlaceFromAccountTraderNotAutorizedSkipListID" status="run" time="1.093781" timestamp="2020-02-19T00:41:29.922917"/>
   <testcase name="PlaceOCOwithOneLeg" status="run" time="0.015604" timestamp="2020-02-19T00:41:31.110451"/>
   <testcase name="RiskForOCO" status="run" time="0.078133" timestamp="2020-02-19T00:41:31.219818"/>
   <testcase name="TwoLegsFilledFully" status="run" time="0.140638" timestamp="2020-02-19T00:41:31.407350"/>
   <testsuite failures="0" name="CheckSendsCancelAfterPlace" skipped="0" tests="2" testsuite_failed="False" time="10.687866" timestamp="2020-02-19T00:41:31.594832">
    <testcase name="LMT_MKT_OCO" status="run" time="5.156441" timestamp="2020-02-19T00:41:31.735450"/>
    <testcase name="MKT_MKT_OCO" status="run" time="5.17207" timestamp="2020-02-19T00:41:37.001253"/>
   </testsuite>
   <testsuite failures="0" name="LegsSizeChangesAfterPartialFill" skipped="0" tests="2" testsuite_failed="False" time="0.689853" timestamp="2020-02-19T00:41:42.360832">
    <testcase name="PartialFillsHandling_Disable" status="run" time="0.156547" timestamp="2020-02-19T00:41:42.532707"/>
    <testcase name="PartialFillsHandling_Enable" status="run" time="0.20312" timestamp="2020-02-19T00:41:42.767086"/>
   </testsuite>
   <testsuite failures="0" name="PlaceModifyOneLegFill" skipped="0" tests="1" testsuite_failed="False" time="0.514496" timestamp="2020-02-19T00:41:43.111986">
    <testcase name="LMT_GTC-LMT_GTD" status="run" time="0.250208" timestamp="2020-02-19T00:41:43.282763"/>
   </testsuite>
   <testsuite failures="0" name="WorkingCompoundAsLeg" skipped="0" tests="2" testsuite_failed="False" time="0.92191" timestamp="2020-02-19T00:41:43.688986">
    <testcase name="AddWOPOtoWOCO1toWOCO2" status="run" time="0.250001" timestamp="2020-02-19T00:41:43.860875"/>
    <testcase name="OCOOPOAcc1Acc2" status="run" time="0.265954" timestamp="2020-02-19T00:41:44.235871"/>
   </testsuite>
   <testsuite failures="0" name="WorkingOrderAsLeg" skipped="0" tests="6" testsuite_failed="False" time="1.734421" timestamp="2020-02-19T00:41:44.689035">
    <testcase name="AddTwiceOneWO" status="run" time="0.141018" timestamp="2020-02-19T00:41:44.829640"/>
    <testcase name="AddTwoWO" status="run" time="0.187587" timestamp="2020-02-19T00:41:45.079667"/>
    <testcase name="AddWOtoWOCO" status="run" time="0.171937" timestamp="2020-02-19T00:41:45.360940"/>
    <testcase name="AddWOtoWOCOFromDiffAcc" status="run" time="0.140633" timestamp="2020-02-19T00:41:45.720309"/>
    <testcase name="OneWorkingOrderAsLeg" status="run" time="0.141041" timestamp="2020-02-19T00:41:45.954713"/>
    <testcase name="TwoWorkingOrdersAsLegs" status="run" time="0.094138" timestamp="2020-02-19T00:41:46.204700"/>
   </testsuite>
  </testsuite>
  <testsuite failures="0" name="OPO" skipped="0" tests="30" testsuite_failed="False" time="45.188982" timestamp="2020-02-19T00:41:46.532858">
   <testcase name="AddTwoNewOrders" status="run" time="0.109045" timestamp="2020-02-19T00:41:46.658170"/>
   <testcase name="DurationModificationForOPO" status="run" time="1.076023" timestamp="2020-02-19T00:41:46.863104"/>
   <testcase name="ModifyOPOSecLegAfterFillPriLeg" status="run" time="0.156566" timestamp="2020-02-19T00:41:48.048510"/>
   <testcase name="OCOasPRILegInOPO_PFH_Disable" status="run" time="0.093738" timestamp="2020-02-19T00:41:48.314161"/>
   <testcase name="OCOasPRILegInOPO_PFH_Enable" status="run" time="0.078131" timestamp="2020-02-19T00:41:48.532904"/>
   <testcase name="OPOPartialFill" status="run" time="0.187526" timestamp="2020-02-19T00:41:48.720413"/>
   <testcase name="OPOwith1LMT_9MKT_FillLMT" status="run" time="0.671881" timestamp="2020-02-19T00:41:49.032939"/>
   <testcase name="PlaceFromAccountTraderNotAutorized" status="run" time="0.03137" timestamp="2020-02-19T00:41:49.829810"/>
   <testcase name="PlaceFromAccountTraderNotAutorizedSkipListID" status="run" time="0.890661" timestamp="2020-02-19T00:41:49.970451"/>
   <testcase name="PlaceOPO2legsActiveAtCancelPRI" status="run" time="10.109722" timestamp="2020-02-19T00:41:51.001738"/>
   <testcase name="PlaceOPO3legsCancelSecLegs" status="run" time="0.093745" timestamp="2020-02-19T00:42:01.220834"/>
   <testcase name="PlaceOPO4legsCancelPRI" status="run" time="0.124968" timestamp="2020-02-19T00:42:01.424002"/>
   <testcase name="PlaceOPO4legsRejectPRI" status="run" time="0.109373" timestamp="2020-02-19T00:42:01.627094"/>
   <testcase name="PlaceOPOwithOneLeg" status="run" time="0.031299" timestamp="2020-02-19T00:42:01.830229"/>
   <testcase name="PlaceOPO_L0_OCO_L1_OCO" status="run" time="0.04688" timestamp="2020-02-19T00:42:01.955228"/>
   <testcase name="PLegFilledPartialyPLegFilledFully_LMT-MKT" status="run" time="5.172061" timestamp="2020-02-19T00:42:02.111471"/>
   <testcase name="RiskForOPO" status="run" time="0.078121" timestamp="2020-02-19T00:42:07.408537"/>
   <testsuite failures="0" name="LegsSizeChangesAfterPartialFill" skipped="0" tests="9" testsuite_failed="False" time="22.735167" timestamp="2020-02-19T00:42:07.549161">
    <testcase name="PRIFillPartFillFull_PFH_Enable" status="run" time="5.218897" timestamp="2020-02-19T00:42:07.705439"/>
    <testcase name="PRILegFill_1_1_PFH_Enable" status="run" time="5.1407869999999996" timestamp="2020-02-19T00:42:13.018111"/>
    <testcase name="PRIModTo10PRIFill1" status="run" time="5.172057" timestamp="2020-02-19T00:42:18.299527"/>
    <testcase name="PRIModTo10PRIFill5Fill5" status="run" time="0.156573" timestamp="2020-02-19T00:42:23.612216"/>
    <testcase name="PRIModTo20PRIFillPart3Time" status="run" time="5.218947" timestamp="2020-02-19T00:42:23.909081"/>
    <testcase name="PRIsize10Modto5Fill5" status="run" time="0.152187" timestamp="2020-02-19T00:42:29.257083"/>
    <testcase name="PRIsize20Modto40Fill20" status="run" time="0.124985" timestamp="2020-02-19T00:42:29.534290"/>
    <testcase name="SecModTo11PRIFillPartFillFull_PFH_Enable" status="run" time="0.171839" timestamp="2020-02-19T00:42:29.737443"/>
    <testcase name="SecModTo30PRIFillPartFillFull_PFH_Enable" status="run" time="0.156682" timestamp="2020-02-19T00:42:30.018952"/>
   </testsuite>
   <testsuite failures="0" name="PlaceModifyOneLegFill" skipped="0" tests="1" testsuite_failed="False" time="0.453152" timestamp="2020-02-19T00:42:30.331184">
    <testcase name="LMT_GTC-LMT_GTD" status="run" time="0.125497" timestamp="2020-02-19T00:42:30.503066"/>
   </testsuite>
   <testsuite failures="0" name="WorkingOrderAsLeg" skipped="0" tests="3" testsuite_failed="False" time="0.76566" timestamp="2020-02-19T00:42:30.846816">
    <testcase name="AddWOtoWOPO" status="run" time="0.093732" timestamp="2020-02-19T00:42:31.018713"/>
    <testcase name="CancelSecLegWOPOaddWOtoWOPO" status="run" time="0.10971" timestamp="2020-02-19T00:42:31.206200"/>
    <testcase name="TwoSTPWOasLegs" status="run" time="0.078114" timestamp="2020-02-19T00:42:31.409342"/>
   </testsuite>
  </testsuite>
  <testsuite failures="6" name="PartialQty" skipped="0" tests="6" testsuite_failed="True" time="2.046954" timestamp="2020-02-19T00:42:31.799992">
   <testcase name="Bracket_with_no_PFH" status="run" time="0.031556" timestamp="2020-02-19T00:42:32.987524">
    <failure message="Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\PartialQty\Bracket_with_no_PFH_tc.py&quot;, line 17, in &lt;module&gt;
    OrderActionUtils.CreateOrder(pqContract, ModuleAgent.OrderSide.Buy, 2, ModuleAgent.OrderType.Limit, limitPrice=100),
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\OrderActionUtils.py&quot;, line 620, in CreateOrder
    if minVisibleSize is not None:
NameError: name 'minVisibleSize' is not defined
">Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\PartialQty\Bracket_with_no_PFH_tc.py&quot;, line 17, in &lt;module&gt;
    OrderActionUtils.CreateOrder(pqContract, ModuleAgent.OrderSide.Buy, 2, ModuleAgent.OrderType.Limit, limitPrice=100),
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\OrderActionUtils.py&quot;, line 620, in CreateOrder
    if minVisibleSize is not None:
NameError: name 'minVisibleSize' is not defined
</failure>
   </testcase>
   <testcase name="Bracket_with_PFH" status="run" time="0.031229" timestamp="2020-02-19T00:42:33.112555">
    <failure message="Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\PartialQty\Bracket_with_PFH_tc.py&quot;, line 16, in &lt;module&gt;
    OrderActionUtils.CreateOrder(pqContract, ModuleAgent.OrderSide.Buy, 0.02, ModuleAgent.OrderType.Limit, limitPrice=100),
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\OrderActionUtils.py&quot;, line 620, in CreateOrder
    if minVisibleSize is not None:
NameError: name 'minVisibleSize' is not defined
">Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\PartialQty\Bracket_with_PFH_tc.py&quot;, line 16, in &lt;module&gt;
    OrderActionUtils.CreateOrder(pqContract, ModuleAgent.OrderSide.Buy, 0.02, ModuleAgent.OrderType.Limit, limitPrice=100),
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\OrderActionUtils.py&quot;, line 620, in CreateOrder
    if minVisibleSize is not None:
NameError: name 'minVisibleSize' is not defined
</failure>
   </testcase>
   <testcase name="OCO_with_no_PFH" status="run" time="0.015608" timestamp="2020-02-19T00:42:33.253163">
    <failure message="Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\PartialQty\OCO_with_no_PFH_tc.py&quot;, line 14, in &lt;module&gt;
    OrderActionUtils.CreateOrder(pqContract, ModuleAgent.OrderSide.Buy, 0.0009, ModuleAgent.OrderType.Limit, limitPrice=100),
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\OrderActionUtils.py&quot;, line 620, in CreateOrder
    if minVisibleSize is not None:
NameError: name 'minVisibleSize' is not defined
">Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\PartialQty\OCO_with_no_PFH_tc.py&quot;, line 14, in &lt;module&gt;
    OrderActionUtils.CreateOrder(pqContract, ModuleAgent.OrderSide.Buy, 0.0009, ModuleAgent.OrderType.Limit, limitPrice=100),
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\OrderActionUtils.py&quot;, line 620, in CreateOrder
    if minVisibleSize is not None:
NameError: name 'minVisibleSize' is not defined
</failure>
   </testcase>
   <testcase name="OCO_with_PFH" status="run" time="0.031286" timestamp="2020-02-19T00:42:33.378160">
    <failure message="Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\PartialQty\OCO_with_PFH_tc.py&quot;, line 15, in &lt;module&gt;
    OrderActionUtils.CreateOrder(epContract, ModuleAgent.OrderSide.Buy, 20, ModuleAgent.OrderType.Limit, limitPrice=100),
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\OrderActionUtils.py&quot;, line 620, in CreateOrder
    if minVisibleSize is not None:
NameError: name 'minVisibleSize' is not defined
">Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\PartialQty\OCO_with_PFH_tc.py&quot;, line 15, in &lt;module&gt;
    OrderActionUtils.CreateOrder(epContract, ModuleAgent.OrderSide.Buy, 20, ModuleAgent.OrderType.Limit, limitPrice=100),
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\OrderActionUtils.py&quot;, line 620, in CreateOrder
    if minVisibleSize is not None:
NameError: name 'minVisibleSize' is not defined
</failure>
   </testcase>
   <testcase name="OPO_with_no_PFH" status="run" time="0.031577" timestamp="2020-02-19T00:42:33.534428">
    <failure message="Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\PartialQty\OPO_with_no_PFH_tc.py&quot;, line 15, in &lt;module&gt;
    OrderActionUtils.CreateOrder(pqContract, ModuleAgent.OrderSide.Sell, 0.0001, ModuleAgent.OrderType.Limit, limitPrice=100),
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\OrderActionUtils.py&quot;, line 620, in CreateOrder
    if minVisibleSize is not None:
NameError: name 'minVisibleSize' is not defined
">Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\PartialQty\OPO_with_no_PFH_tc.py&quot;, line 15, in &lt;module&gt;
    OrderActionUtils.CreateOrder(pqContract, ModuleAgent.OrderSide.Sell, 0.0001, ModuleAgent.OrderType.Limit, limitPrice=100),
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\OrderActionUtils.py&quot;, line 620, in CreateOrder
    if minVisibleSize is not None:
NameError: name 'minVisibleSize' is not defined
</failure>
   </testcase>
   <testcase name="OPO_with_PFH" status="run" time="0.015637" timestamp="2020-02-19T00:42:33.690659">
    <failure message="Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\PartialQty\OPO_with_PFH_tc.py&quot;, line 14, in &lt;module&gt;
    OrderActionUtils.CreateOrder(pqContract, ModuleAgent.OrderSide.Buy, 0.02, ModuleAgent.OrderType.Limit, limitPrice=100),
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\OrderActionUtils.py&quot;, line 620, in CreateOrder
    if minVisibleSize is not None:
NameError: name 'minVisibleSize' is not defined
">Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\PartialQty\OPO_with_PFH_tc.py&quot;, line 14, in &lt;module&gt;
    OrderActionUtils.CreateOrder(pqContract, ModuleAgent.OrderSide.Buy, 0.02, ModuleAgent.OrderType.Limit, limitPrice=100),
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\OrderActionUtils.py&quot;, line 620, in CreateOrder
    if minVisibleSize is not None:
NameError: name 'minVisibleSize' is not defined
</failure>
   </testcase>
  </testsuite>
  <testsuite failures="6" name="RiskCompoundOrderMargin" skipped="0" tests="6" testsuite_failed="True" time="1.671866" timestamp="2020-02-19T00:42:33.925126">
   <testcase name="01_SecondaryOCO_Simple" status="run" time="0.078458" timestamp="2020-02-19T00:42:34.159443">
    <failure message="Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\RiskCompoundOrderMargin\01_SecondaryOCO_Simple.tc.py&quot;, line 55, in &lt;module&gt;
    test(&quot;SecondaryOCO_Simple&quot;)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\RiskCompoundOrderMargin\01_SecondaryOCO_Simple.tc.py&quot;, line 29, in test
    ModifyCompoundLegOrderRejected(context1, OCOEvent, leg22, rejectReason, newQuantity = 2)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\SystemAreas\OrderProcessing\OrderRouting\Suites\CompoundOrders\PythonModules\CompoundActions.py&quot;, line 283, in ModifyCompoundLegOrderRejected
    rejectReason = rejectReason, isFailure = isFailure)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\SimpleOrderActions.py&quot;, line 772, in AssertModifyOrderRejected
    expectedOEAttrs=expectedOEAttrs, visibleSize=visibleSize)
TypeError: __assertModifyOrderRejected() got an unexpected keyword argument 'visibleSize'
">Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\RiskCompoundOrderMargin\01_SecondaryOCO_Simple.tc.py&quot;, line 55, in &lt;module&gt;
    test(&quot;SecondaryOCO_Simple&quot;)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\RiskCompoundOrderMargin\01_SecondaryOCO_Simple.tc.py&quot;, line 29, in test
    ModifyCompoundLegOrderRejected(context1, OCOEvent, leg22, rejectReason, newQuantity = 2)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\SystemAreas\OrderProcessing\OrderRouting\Suites\CompoundOrders\PythonModules\CompoundActions.py&quot;, line 283, in ModifyCompoundLegOrderRejected
    rejectReason = rejectReason, isFailure = isFailure)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\SimpleOrderActions.py&quot;, line 772, in AssertModifyOrderRejected
    expectedOEAttrs=expectedOEAttrs, visibleSize=visibleSize)
TypeError: __assertModifyOrderRejected() got an unexpected keyword argument 'visibleSize'
</failure>
   </testcase>
   <testcase name="02_SecondaryOPO_Simple" status="run" time="0.093749" timestamp="2020-02-19T00:42:34.378203">
    <failure message="Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\RiskCompoundOrderMargin\02_SecondaryOPO_Simple.tc.py&quot;, line 72, in &lt;module&gt;
    test(caseName = &quot;SecondaryOPO_Simple&quot;)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\RiskCompoundOrderMargin\02_SecondaryOPO_Simple.tc.py&quot;, line 31, in test
    ModifyCompoundLegOrderRejected(context1, OPOEvent, leg1, rejectReason, newQuantity = 2)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\SystemAreas\OrderProcessing\OrderRouting\Suites\CompoundOrders\PythonModules\CompoundActions.py&quot;, line 283, in ModifyCompoundLegOrderRejected
    rejectReason = rejectReason, isFailure = isFailure)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\SimpleOrderActions.py&quot;, line 772, in AssertModifyOrderRejected
    expectedOEAttrs=expectedOEAttrs, visibleSize=visibleSize)
TypeError: __assertModifyOrderRejected() got an unexpected keyword argument 'visibleSize'
">Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\RiskCompoundOrderMargin\02_SecondaryOPO_Simple.tc.py&quot;, line 72, in &lt;module&gt;
    test(caseName = &quot;SecondaryOPO_Simple&quot;)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\RiskCompoundOrderMargin\02_SecondaryOPO_Simple.tc.py&quot;, line 31, in test
    ModifyCompoundLegOrderRejected(context1, OPOEvent, leg1, rejectReason, newQuantity = 2)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\SystemAreas\OrderProcessing\OrderRouting\Suites\CompoundOrders\PythonModules\CompoundActions.py&quot;, line 283, in ModifyCompoundLegOrderRejected
    rejectReason = rejectReason, isFailure = isFailure)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\SimpleOrderActions.py&quot;, line 772, in AssertModifyOrderRejected
    expectedOEAttrs=expectedOEAttrs, visibleSize=visibleSize)
TypeError: __assertModifyOrderRejected() got an unexpected keyword argument 'visibleSize'
</failure>
   </testcase>
   <testcase name="03_PrimaryOCO_Compounds" status="run" time="0.171866" timestamp="2020-02-19T00:42:34.596968">
    <failure message="Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\RiskCompoundOrderMargin\03_PrimaryOCO_Compounds.tc.py&quot;, line 107, in &lt;module&gt;
    test(&quot;PrimaryOCO_Compounds&quot;)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\RiskCompoundOrderMargin\03_PrimaryOCO_Compounds.tc.py&quot;, line 52, in test
    ModifyCompoundLegOrderRejected(context1, OPOEvent2, leg22, rejectReason, newQuantity = 2)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\SystemAreas\OrderProcessing\OrderRouting\Suites\CompoundOrders\PythonModules\CompoundActions.py&quot;, line 283, in ModifyCompoundLegOrderRejected
    rejectReason = rejectReason, isFailure = isFailure)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\SimpleOrderActions.py&quot;, line 772, in AssertModifyOrderRejected
    expectedOEAttrs=expectedOEAttrs, visibleSize=visibleSize)
TypeError: __assertModifyOrderRejected() got an unexpected keyword argument 'visibleSize'
">Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\RiskCompoundOrderMargin\03_PrimaryOCO_Compounds.tc.py&quot;, line 107, in &lt;module&gt;
    test(&quot;PrimaryOCO_Compounds&quot;)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\RiskCompoundOrderMargin\03_PrimaryOCO_Compounds.tc.py&quot;, line 52, in test
    ModifyCompoundLegOrderRejected(context1, OPOEvent2, leg22, rejectReason, newQuantity = 2)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\SystemAreas\OrderProcessing\OrderRouting\Suites\CompoundOrders\PythonModules\CompoundActions.py&quot;, line 283, in ModifyCompoundLegOrderRejected
    rejectReason = rejectReason, isFailure = isFailure)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\SimpleOrderActions.py&quot;, line 772, in AssertModifyOrderRejected
    expectedOEAttrs=expectedOEAttrs, visibleSize=visibleSize)
TypeError: __assertModifyOrderRejected() got an unexpected keyword argument 'visibleSize'
</failure>
   </testcase>
   <testcase name="04_PrimaryOPO_Compounds" status="run" time="0.109364" timestamp="2020-02-19T00:42:34.893834">
    <failure message="Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\RiskCompoundOrderMargin\04_PrimaryOPO_Compounds.tc.py&quot;, line 94, in &lt;module&gt;
    test(caseName = &quot;PrimaryOPO_Compounds&quot;)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\RiskCompoundOrderMargin\04_PrimaryOPO_Compounds.tc.py&quot;, line 40, in test
    ModifyCompoundLegOrderRejected(context1, OPOEvent1, leg11, rejectReason, newQuantity = 2)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\SystemAreas\OrderProcessing\OrderRouting\Suites\CompoundOrders\PythonModules\CompoundActions.py&quot;, line 283, in ModifyCompoundLegOrderRejected
    rejectReason = rejectReason, isFailure = isFailure)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\SimpleOrderActions.py&quot;, line 772, in AssertModifyOrderRejected
    expectedOEAttrs=expectedOEAttrs, visibleSize=visibleSize)
TypeError: __assertModifyOrderRejected() got an unexpected keyword argument 'visibleSize'
">Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\RiskCompoundOrderMargin\04_PrimaryOPO_Compounds.tc.py&quot;, line 94, in &lt;module&gt;
    test(caseName = &quot;PrimaryOPO_Compounds&quot;)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\RiskCompoundOrderMargin\04_PrimaryOPO_Compounds.tc.py&quot;, line 40, in test
    ModifyCompoundLegOrderRejected(context1, OPOEvent1, leg11, rejectReason, newQuantity = 2)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\SystemAreas\OrderProcessing\OrderRouting\Suites\CompoundOrders\PythonModules\CompoundActions.py&quot;, line 283, in ModifyCompoundLegOrderRejected
    rejectReason = rejectReason, isFailure = isFailure)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\SimpleOrderActions.py&quot;, line 772, in AssertModifyOrderRejected
    expectedOEAttrs=expectedOEAttrs, visibleSize=visibleSize)
TypeError: __assertModifyOrderRejected() got an unexpected keyword argument 'visibleSize'
</failure>
   </testcase>
   <testcase name="05_MultipleSecondaries" status="run" time="0.062489" timestamp="2020-02-19T00:42:35.143884">
    <failure message="Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\RiskCompoundOrderMargin\05_MultipleSecondaries.tc.py&quot;, line 88, in &lt;module&gt;
    test(caseName = &quot;OPO_MultipleSecondaries&quot;)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\RiskCompoundOrderMargin\05_MultipleSecondaries.tc.py&quot;, line 28, in test
    ModifyCompoundLegOrderRejected(context1, OPOEvent, leg1, rejectReason, newQuantity = 2)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\SystemAreas\OrderProcessing\OrderRouting\Suites\CompoundOrders\PythonModules\CompoundActions.py&quot;, line 283, in ModifyCompoundLegOrderRejected
    rejectReason = rejectReason, isFailure = isFailure)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\SimpleOrderActions.py&quot;, line 772, in AssertModifyOrderRejected
    expectedOEAttrs=expectedOEAttrs, visibleSize=visibleSize)
TypeError: __assertModifyOrderRejected() got an unexpected keyword argument 'visibleSize'
">Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\RiskCompoundOrderMargin\05_MultipleSecondaries.tc.py&quot;, line 88, in &lt;module&gt;
    test(caseName = &quot;OPO_MultipleSecondaries&quot;)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\RiskCompoundOrderMargin\05_MultipleSecondaries.tc.py&quot;, line 28, in test
    ModifyCompoundLegOrderRejected(context1, OPOEvent, leg1, rejectReason, newQuantity = 2)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\SystemAreas\OrderProcessing\OrderRouting\Suites\CompoundOrders\PythonModules\CompoundActions.py&quot;, line 283, in ModifyCompoundLegOrderRejected
    rejectReason = rejectReason, isFailure = isFailure)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\SimpleOrderActions.py&quot;, line 772, in AssertModifyOrderRejected
    expectedOEAttrs=expectedOEAttrs, visibleSize=visibleSize)
TypeError: __assertModifyOrderRejected() got an unexpected keyword argument 'visibleSize'
</failure>
   </testcase>
   <testcase name="06_AddingOrders" status="run" time="0.109385" timestamp="2020-02-19T00:42:35.331349">
    <failure message="Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\RiskCompoundOrderMargin\06_AddingOrders.tc.py&quot;, line 97, in &lt;module&gt;
    test(caseName = &quot;OPO_AddingOrders&quot;)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\RiskCompoundOrderMargin\06_AddingOrders.tc.py&quot;, line 31, in test
    ModifyCompoundLegOrderRejected(context1, OPOEvent, leg1, rejectReason, newQuantity = 2)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\SystemAreas\OrderProcessing\OrderRouting\Suites\CompoundOrders\PythonModules\CompoundActions.py&quot;, line 283, in ModifyCompoundLegOrderRejected
    rejectReason = rejectReason, isFailure = isFailure)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\SimpleOrderActions.py&quot;, line 772, in AssertModifyOrderRejected
    expectedOEAttrs=expectedOEAttrs, visibleSize=visibleSize)
TypeError: __assertModifyOrderRejected() got an unexpected keyword argument 'visibleSize'
">Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\RiskCompoundOrderMargin\06_AddingOrders.tc.py&quot;, line 97, in &lt;module&gt;
    test(caseName = &quot;OPO_AddingOrders&quot;)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\RiskCompoundOrderMargin\06_AddingOrders.tc.py&quot;, line 31, in test
    ModifyCompoundLegOrderRejected(context1, OPOEvent, leg1, rejectReason, newQuantity = 2)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\SystemAreas\OrderProcessing\OrderRouting\Suites\CompoundOrders\PythonModules\CompoundActions.py&quot;, line 283, in ModifyCompoundLegOrderRejected
    rejectReason = rejectReason, isFailure = isFailure)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\SimpleOrderActions.py&quot;, line 772, in AssertModifyOrderRejected
    expectedOEAttrs=expectedOEAttrs, visibleSize=visibleSize)
TypeError: __assertModifyOrderRejected() got an unexpected keyword argument 'visibleSize'
</failure>
   </testcase>
  </testsuite>
  <testsuite failures="3" name="Suspend" skipped="0" tests="19" testsuite_failed="True" time="14.750258" timestamp="2020-02-19T00:42:35.659738">
   <testcase name="ActivateAtLegs_OCO" status="run" time="0.078128" timestamp="2020-02-19T00:42:35.831365"/>
   <testcase name="ActivateAtLegs_OPO" status="run" time="0.046878" timestamp="2020-02-19T00:42:36.018875"/>
   <testcase name="Bracket_Leg1_OCO" status="run" time="0.078113" timestamp="2020-02-19T00:42:36.190757">
    <failure message="Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\Suspend\Bracket_Leg1_OCO.tc.py&quot;, line 36, in &lt;module&gt;
    newActtime = GetTimeFromNow(hours = 1), isFailure = True)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\SystemAreas\OrderProcessing\OrderRouting\Suites\CompoundOrders\PythonModules\CompoundActions.py&quot;, line 283, in ModifyCompoundLegOrderRejected
    rejectReason = rejectReason, isFailure = isFailure)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\SimpleOrderActions.py&quot;, line 772, in AssertModifyOrderRejected
    expectedOEAttrs=expectedOEAttrs, visibleSize=visibleSize)
TypeError: __assertModifyOrderRejected() got an unexpected keyword argument 'visibleSize'
">Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\Suspend\Bracket_Leg1_OCO.tc.py&quot;, line 36, in &lt;module&gt;
    newActtime = GetTimeFromNow(hours = 1), isFailure = True)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\SystemAreas\OrderProcessing\OrderRouting\Suites\CompoundOrders\PythonModules\CompoundActions.py&quot;, line 283, in ModifyCompoundLegOrderRejected
    rejectReason = rejectReason, isFailure = isFailure)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\SimpleOrderActions.py&quot;, line 772, in AssertModifyOrderRejected
    expectedOEAttrs=expectedOEAttrs, visibleSize=visibleSize)
TypeError: __assertModifyOrderRejected() got an unexpected keyword argument 'visibleSize'
</failure>
   </testcase>
   <testcase name="Bracket_Parked_FirstOCOLeg" status="run" time="0.03126" timestamp="2020-02-19T00:42:36.393881"/>
   <testcase name="Bracket_Parked_SecondOCOLeg" status="run" time="0.046869" timestamp="2020-02-19T00:42:36.550155"/>
   <testcase name="ModifyLegs_AAT_APT_inOCO_Autoactivate" status="run" time="2.125035" timestamp="2020-02-19T00:42:36.722054"/>
   <testcase name="ModifyLegs_AAT_APT_inOCO_Autopark" status="run" time="0.078121" timestamp="2020-02-19T00:42:39.018994">
    <failure message="Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\Suspend\ModifyLegs_AAT_APT_inOCO_Autopark.tc.py&quot;, line 36, in &lt;module&gt;
    ModifyCompoundLegOrderRejected(context, OCOEvent, leg2, Reject_AATinThePast, newActtime = GetTimeFromNow(hours = -1))
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\SystemAreas\OrderProcessing\OrderRouting\Suites\CompoundOrders\PythonModules\CompoundActions.py&quot;, line 283, in ModifyCompoundLegOrderRejected
    rejectReason = rejectReason, isFailure = isFailure)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\SimpleOrderActions.py&quot;, line 772, in AssertModifyOrderRejected
    expectedOEAttrs=expectedOEAttrs, visibleSize=visibleSize)
TypeError: __assertModifyOrderRejected() got an unexpected keyword argument 'visibleSize'
">Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\Suspend\ModifyLegs_AAT_APT_inOCO_Autopark.tc.py&quot;, line 36, in &lt;module&gt;
    ModifyCompoundLegOrderRejected(context, OCOEvent, leg2, Reject_AATinThePast, newActtime = GetTimeFromNow(hours = -1))
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\SystemAreas\OrderProcessing\OrderRouting\Suites\CompoundOrders\PythonModules\CompoundActions.py&quot;, line 283, in ModifyCompoundLegOrderRejected
    rejectReason = rejectReason, isFailure = isFailure)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\SimpleOrderActions.py&quot;, line 772, in AssertModifyOrderRejected
    expectedOEAttrs=expectedOEAttrs, visibleSize=visibleSize)
TypeError: __assertModifyOrderRejected() got an unexpected keyword argument 'visibleSize'
</failure>
   </testcase>
   <testcase name="ModifyLegs_AAT_APT_inOPO_Autoactivate" status="run" time="2.171921" timestamp="2020-02-19T00:42:39.222134"/>
   <testcase name="ModifyLegs_AAT_APT_inOPO_Autopark" status="run" time="0.047799" timestamp="2020-02-19T00:42:41.550336">
    <failure message="Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\Suspend\ModifyLegs_AAT_APT_inOPO_Autopark.tc.py&quot;, line 38, in &lt;module&gt;
    ModifyCompoundLegOrderRejected(context, OPOEvent, leg1, Reject_AATinThePast, newActtime = GetTimeFromNow(hours = -1))
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\SystemAreas\OrderProcessing\OrderRouting\Suites\CompoundOrders\PythonModules\CompoundActions.py&quot;, line 283, in ModifyCompoundLegOrderRejected
    rejectReason = rejectReason, isFailure = isFailure)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\SimpleOrderActions.py&quot;, line 772, in AssertModifyOrderRejected
    expectedOEAttrs=expectedOEAttrs, visibleSize=visibleSize)
TypeError: __assertModifyOrderRejected() got an unexpected keyword argument 'visibleSize'
">Traceback (most recent call last):
  File &quot;C:\xpit.com\applications\ScriptFramework\ScriptKit\Core\Invoker.py&quot;, line 29, in invoke_script
    exec(code, isolated_dict)
  File &quot;Scripts/Gateway/SystemAreas/OrderProcessing/OrderRouting/Suites/CompoundOrders\Suspend\ModifyLegs_AAT_APT_inOPO_Autopark.tc.py&quot;, line 38, in &lt;module&gt;
    ModifyCompoundLegOrderRejected(context, OPOEvent, leg1, Reject_AATinThePast, newActtime = GetTimeFromNow(hours = -1))
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\SystemAreas\OrderProcessing\OrderRouting\Suites\CompoundOrders\PythonModules\CompoundActions.py&quot;, line 283, in ModifyCompoundLegOrderRejected
    rejectReason = rejectReason, isFailure = isFailure)
  File &quot;C:\xpit.com\applications\ScriptFramework\Scripts\Gateway\PythonModules\SimpleOrderActions.py&quot;, line 772, in AssertModifyOrderRejected
    expectedOEAttrs=expectedOEAttrs, visibleSize=visibleSize)
TypeError: __assertModifyOrderRejected() got an unexpected keyword argument 'visibleSize'
</failure>
   </testcase>
   <testcase name="OCO_ActiveLegsWithDiff_APT" status="run" time="3.046981" timestamp="2020-02-19T00:42:41.722192"/>
   <testcase name="OCO_ParkLegsWithDiff_AAT" status="run" time="3.046968" timestamp="2020-02-19T00:42:44.925433"/>
   <testcase name="OPO_Parked_AllLegs" status="run" time="0.046875" timestamp="2020-02-19T00:42:48.128655"/>
   <testcase name="OPO_Parked_OnlySecLeg" status="run" time="0.046832" timestamp="2020-02-19T00:42:48.331816"/>
   <testcase name="OPO_SecLegWith_APT_ActivePRILeg" status="run" time="0.062412" timestamp="2020-02-19T00:42:48.472512"/>
   <testcase name="OPO_SecLegWith_APT_ParkPRILeg" status="run" time="0.062506" timestamp="2020-02-19T00:42:48.691177"/>
   <testcase name="ParkActivateBracketOCOLegsAfterFillOPOPriLeg" status="run" time="0.390618" timestamp="2020-02-19T00:42:48.909936"/>
   <testcase name="ParkActivateOPOSecLegAfterFillPriLeg" status="run" time="0.21874" timestamp="2020-02-19T00:42:49.425589"/>
   <testcase name="PartialFillOCO" status="run" time="0.109377" timestamp="2020-02-19T00:42:49.784966"/>
   <testcase name="PartialFillOPO" status="run" time="0.1875" timestamp="2020-02-19T00:42:50.066221"/>
  </testsuite>
  <testsuite failures="0" name="viaOmnibus" skipped="0" tests="5" testsuite_failed="False" time="8.453327" timestamp="2020-02-19T00:42:50.472572">
   <testcase name="BRACKETevents" status="run" time="0.249985" timestamp="2020-02-19T00:42:55.128903"/>
   <testcase name="OCOevents" status="run" time="0.140619" timestamp="2020-02-19T00:42:55.488284"/>
   <testcase name="OPOevents" status="run" time="0.203101" timestamp="2020-02-19T00:42:55.785194"/>
   <testsuite failures="0" name="ExecutionAccountId" skipped="0" tests="2" testsuite_failed="False" time="2.687604" timestamp="2020-02-19T00:42:56.066411">
    <testcase name="ExecutionAccountIDOpoOnInternalOmnibus" status="run" time="0.234383" timestamp="2020-02-19T00:42:58.019601"/>
    <testcase name="ExecutionAccountIDOpoOnOmnibus" status="run" time="0.234398" timestamp="2020-02-19T00:42:58.379008"/>
   </testsuite>
  </testsuite>
 </testsuite>
</testsuites>